README.md

```diff
@@ -19,7 +19,7 @@ make the `xvfb-run` command available): `apt install xvfb`.
 
 ## Run the client
 
-To ensure that file decryption works, please import [this test private key](https://raw.githubusercontent.com/freedomofpress/securedrop/0a901362b84a5378fba80e9cd0ffe4542bdcd598/securedrop/tests/files/test_journalist_key.sec) into your GnuPG keyring. Submissions in the SecureDrop development environment can be decrypted with this test key. 
+To ensure that file decryption works, please import [this test private key](https://raw.githubusercontent.com/freedomofpress/securedrop/0a901362b84a5378fba80e9cd0ffe4542bdcd598/securedrop/tests/files/test_journalist_key.sec) into your GnuPG keyring. Submissions in the SecureDrop development environment can be decrypted with this test key.
```

Comment:
Perhaps this should just point to `develop` and not that specific commit. If you disagree, merge away.

Reply:
Sorry, my editor is configured to trim trailing whitespace, so that edit just came along for the ride. I suggest we merge this PR and address that change separately.